aws-auth-cognito/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=aws-auth-cognito
+POM_NAME=Amplify Framework for Android - Auth
+POM_DESCRIPTION=Amplify Framework for Android - Auth Plugin for Amazon Cognito (Preview)
```

Comment:
We should rip these Preview descriptions out now.

Reply:
Done - took them out of all the categories